book_rental.rb

```diff
@@ -0,0 +1,22 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] I want to kindly suggest that you extract your rental class and place it in a separate file, this way a single file represents a class. :+1: :sparkles: